chainer/functions/array/split_axis.py

```diff
@@ -68,11 +69,14 @@ def split_axis(x, indices_or_sections, axis):
             If it is a 1-D array of sorted integers, it
             indicates the positions where the array is split.
         axis (int): Axis that the input array is split along.
+        force_tuple (bool): If True, this method returns a tuple even when the
+            number of outputs is one.
 
     Returns:
         ``tuple`` or ``Variable``: Tuple of :class:`~chainer.Variable` objects
              if the number of outputs is more than 1 or
              :class:`~chainer.Variable` otherwise.
+             When `force_tuple` is True, return value is always a tuple.
```

Comment:
It gets easier to understand if we add `regardless of the number of outputs` at the end of the sentence.